tests/DebugCodeLens/DebugCodeLensProvider.test.ts

```diff
@@ -33,7 +33,7 @@ jest.mock('vscode', () => {
     end: Position
 
     constructor(start, end) {
-      rangeConstructor(...arguments)
+      rangeConstructor()
```

Comment:
**Question**: Should we pass the `start` and `end` here? I don't remember what I was thinking 3 years ago. 😆 I'm assuming the tests pass without it which also makes me worry. Heh.

Reply:
see line 4, it got mocked, I didn't dig deeper to see why. When eslint is complaining about using `arguments`, I realized it is a mocked function and we don't check the arguments, so no need to pass anything, thus removed. If it is intended to test the actual constructor, then the original test is not doing that...